tests/models/test_onnx.py

```diff
@@ -84,8 +98,9 @@ def test_error_if_no_input(tmpdir):
     """Test that an exception is thrown when there is no input tensor"""
     model = EvalModelTemplate()
     model.example_input_array = None
-    file_path = os.path.join(tmpdir, "model.onxx")
-    with pytest.raises(ValueError, match=r'`input_sample` and `example_input_array` tensors are both missing'):
+    file_path = os.path.join(tmpdir, "model.onnx")
+    with pytest.raises(ValueError, match=r'Could not export to ONNX since neither input_sample nor '
+                                         r'model.example_input_array attribute is set.'):
```

Comment:
```suggestion
    with pytest.raises(ValueError, match=r'Could not export to ONNX since neither `input_sample` nor'
                                         r' `model.example_input_array` attribute is set.'):
```